.github/issue_template.md

```diff
@@ -0,0 +1,22 @@
+Hi! Before opening a new issue, please make sure it has not already been
```

Comment:
I don't think we need this file for now? We don't use GH issues very heavily

Reply:
Are we keeping this file or removing it?

Reply:
sorry i missed it